README.md

```diff
@@ -115,19 +115,18 @@ In the end, there are no hard and fast rules for when a contribution has to be a
 
 ## Contributors ✨
 
-Thanks goes to these wonderful people ([emoji key][emojis]):
+Thanks goes to these wonderful people ([emoji key][https://github.com/all-contributors/all-contributors#emoji-key]):
```

Comment:
should be  `([emoji key](https://github.com/all-contributors/all-contributors#emoji-key))`